src/Components/Blazor/Blazor/src/Services/WebAssemblyUriHelper.cs

```diff
@@ -40,8 +36,7 @@ protected override void InitializeState()
             // client-side (Mono) use, so it's OK to rely on synchronicity here.
             var baseUri = WebAssemblyJSRuntime.Instance.Invoke<string>(Interop.GetBaseUri);
             var uri = WebAssemblyJSRuntime.Instance.Invoke<string>(Interop.GetLocationHref);
-            SetAbsoluteBaseUri(baseUri);
-            SetAbsoluteUri(uri);
+            base.InitializeState(uri, baseUri);
```

Comment:
Doesn't need to be prefixed with `base.`, does it?

Reply:
done